tests/test_concentintegration.py

```diff
@@ -94,13 +136,13 @@ def test_forced_payment(self):
         assert len(forced_payments) == 1
         assert forced_payments[0].requestor == requestor
         assert forced_payments[0].provider == provider
-        assert forced_payments[0].amount == value
+        assert forced_payments[0].amount == reimburse_amount
         assert forced_payments[0].closure_time == closure_time
 
         assert len(events) == 1
         assert events[0].requestor == requestor
         assert events[0].provider == provider
-        assert events[0].amount == value
+        assert events[0].amount == reimburse_amount
         assert events[0].closure_time == closure_time
 
     def test_forced_subtask_payment(self):
```

Comment:
general remark - it would be better if the assertions here were separated into their own tests - iow, one assertion per test method - that way they're 1) easier to extend an maintain, 2) a single failure doesn't mask more tests...